src/main/scala/zio/redis/api/Strings.scala

```diff
@@ -20,9 +42,20 @@ trait Strings {
   final val mset        = Command("MSET", NonEmptyList(Tuple2(StringInput, ByteInput)), UnitOutput)
   final val msetnx      = Command("MSETNX", NonEmptyList(Tuple2(StringInput, ByteInput)), BoolOutput)
   final val psetex      = Command("PSETEX", Tuple3(StringInput, DurationInput, ByteInput), UnitOutput)
-  final val setbit      = Command("SETBIT", Tuple3(StringInput, LongInput, ByteInput), ByteOutput)
-  final val setex       = Command("SETEX", Tuple3(StringInput, DurationInput, ByteInput), UnitOutput)
-  final val setnx       = Command("SETNX", Tuple2(StringInput, ByteInput), BoolOutput)
-  final val setrange    = Command("SETRANGE", Tuple3(StringInput, LongInput, ByteInput), LongOutput)
-  final val strlen      = Command("STRLEN", StringInput, LongOutput)
+  final val set = Command(
+    "SET",
+    Tuple5(
+      StringInput,
+      StringInput,
+      OptionalInput(StringExpirationInput),
+      OptionalInput(UpdatesInput),
+      OptionalInput(StringKeepTimeToLiveInput)
```

Comment:
This one (and similar) can be modeled by boolean input with some default. Leave it for now, we'll see how the API looks later on!